boot/core/src/boot/task/built_in.clj

```diff
@@ -575,7 +575,7 @@
         action  (partial helpers/sift-action v?)
         process (reduce-kv #(comp (action %2 %3) %1) identity *opts*)]
     (core/with-pre-wrap [fs]
-      (util/info "Sifting output files...\n")
+      (util/dbug* "Sifting with opts: %s\n" (util/pp-str (assoc *opts* :invert v?)))
```

Comment:
I think personally I would rather still have it print the output by default rather than only on debug.  It's a common behavior for most built-in tasks to print something indicating which task was run, and it helps as a base level sanity check that the tasks you think are running are actually happening.

Big 👍  to making the output more meaningful!